pzsd_bot/settings.py

```diff
@@ -15,12 +15,20 @@ class EnvSettings(BaseSettings, env_file=".env", extra="ignore"):
 
 class _Bot(EnvSettings):
     token: str
-    log_level: str = "INFO"
 
 
 Bot = _Bot()
 
 
+class _Miscellaneous(EnvSettings):
+    debug: bool = False
+
+
```

Comment:
Do we need a class for this?

Reply:
i admit it's a little silly to create a class just for this one setting. Thought I was fixing something by moving it out of the `_Bot` class, but it can actually stay there. Suppose I could move it back. Does indeed need to be in a class so that it can be overridden by the `.env` file though.

Reply:
Maybe global settings go in Bot or maybe just rename this class

Reply:
Yea I think I'll just put it back in bot. Maybe if there were more global settings I could keep it here and rename the class to `_Global` or something, but I don't think it feels out of place in `_Bot` honestly.